src/PowerShell/Microsoft.WinGet.Client.Cmdlets/Acl/WinGetAssemblyLoadContext.cs

```diff
@@ -18,9 +18,13 @@ namespace Microsoft.WinGet.Client.Acl
     /// </summary>
     internal class WinGetAssemblyLoadContext : AssemblyLoadContext
     {
-        private static readonly string ExpectedDependencyPath = Path.Combine(
+        private static readonly string SharedDependencyPath = Path.Combine(
             Path.GetDirectoryName(typeof(WinGetAssemblyLoadContext).Assembly.Location),
-            "Dependencies");
+            "SharedDependencies");
+
+        private static readonly string DirectDependencyPath = Path.Combine(
+            Path.GetDirectoryName(typeof(WinGetAssemblyLoadContext).Assembly.Location),

```

Comment:
It looks like you inverted the name suggestions that I used (completely, so functionally it should work).  To me, the "shared" dependencies are the ones that might conflict (and so need to load from the ALC `Load`) and the "direct" ones are the ones that are directly dependencies of the main DLL (and so need to use `Resolving`).  Feel free to use whatever names you want, other than reversing my suggestion which is confusing to me. 🫠

Reply:
🫠